app/views/users/index.html.erb

```diff
@@ -1,2 +1,13 @@
-<h1>Users#index</h1>
-<h2>This is the Users index page</h2>
\ No newline at end of file
+<div class="main-container">
+	<% @users.each do |user| %>
+		<div class="users-wrapper flex">
+			<div class="img-cont">
+				<%# <%= image_tag user.photo, :size => "50x50", alt: "user photo" %> %>
+			</div>
+			<div class="user-details">
+				<h2><%= user.name %></h2>
+				<p>Number of posts: <%= user.posts_counter %></p>
+			</div>
+		</div>
+	<%end%>
+</div>
```

Comment:
- Hey, the index page for your users doesn't seem to show up anything, it's blank even after I seed your database with some data. please, try to fix this then you can add the form functions properly. Please, ensure all your views pages are working correctly then you can proceed to add the form interaction.


![image](https://github.com/Abe1able/Bloggingly/assets/59397159/60baa518-6188-48d9-97d9-9c93d1be1cb8)

 - For instance, I add a user with the id of 1, it throws an error. please, try to fix your controllers so you can implement the form functionality.

![image](https://github.com/Abe1able/Bloggingly/assets/59397159/fcf633a8-70cd-4410-9ab2-b8bc213887a4)

Reply:
I've made sure it works correctly. Here is the screenshot.

![Bloggingly4](https://github.com/Abe1able/Bloggingly/assets/87712877/d31aa90a-02ad-4fb3-b59f-fb93a8c76c74)